metabase/metabase
# Migrate segment revision history to MBQL lib

Epic https://github.com/metabase/metabase/issues/50336

How to verify:
- Create a segment and edit it multiple times with different filters
- Verify that the segment revision history is displayed correctly both in admin settings and `reference` pages

<img width="1395" alt="Screenshot 2024-11-21 at 14 12 16" src="https://github.com/user-attachments/assets/924b03d6-dc7c-4daf-9829-387307fbbcd9">
<img width="1728" alt="Screenshot 2024-11-21 at 14 12 29" src="https://github.com/user-attachments/assets/c5856206-3949-4ed1-bc2e-fa79b5504f23">
<img width="1619" alt="Screenshot 2024-11-21 at 14 12 48" src="https://github.com/user-attachments/assets/fca9832f-4455-4e76-be51-3ba1ef0a30b4">
